assets/admin_themes/Sea_Green/layout/body.scss

```
@@ -5,4 +5,8 @@ body {
     display          : flex;
     min-height       : 100vh;
     flex-direction   : column;
+}
+.modal-header .btn-close:focus {
+    outline : 2px solid currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)